api/app/reviews/api.py

```diff
@@ -431,11 +442,10 @@ class ReviewAssignmentAPI(GetReviewAssignmentMixin, PostReviewAssignmentMixin, r
         'reviews_completed': fields.Integer
     }
 
-    @auth_required
+    @event_admin_or_action_editor_required
     @marshal_with(reviews_count_fields)
-    def get(self):
+    def get(self, event_id):
         args = self.get_req_parser.parse_args()
-        event_id = args['event_id']
         user_id = g.current_user['id']
 
         current_user = user_repository.get_by_id(user_id)

```

Comment:
Now that you've added the decorator, you can remove the current_user.is_event_admin check